src/components/input/input.scss

```diff
@@ -34,7 +34,8 @@ $md-input-underline-disabled-background-image: linear-gradient(to right,
   visibility: visible;
   padding-bottom: 5px;
   transform: translateY(-100%) scale(0.75);
-
+  width: 133%; // 100/75 of the original space now available
```

Comment:
What I'm saying is that this is a good time to move the `0.75` out into a variable and use that variable in the width calculation. This will clean up stuff and make it clear what the intent is. The comment `100 / 75` doesn't tell us anything about where `75` came from. Nor what is the purpose of 75.

Also, if we change the `0.75` scale, we can't possibly know what else to change.